Apps/Sandcastle/gallery/Camera Limiter.html

```diff
@@ -0,0 +1,168 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, minimum-scale=1, user-scalable=no">
+    <meta name="description" content="Use the Camera's bounding object to restrict the camera's movement.">
+    <meta name="cesium-sandcastle-labels" content="Showcases">
+    <title>Cesium Demo</title>
+    <script type="text/javascript" src="../Sandcastle-header.js"></script>
+    <script type="text/javascript" src="../../../ThirdParty/requirejs-2.1.20/require.js"></script>
+    <script type="text/javascript">
+    require.config({
+        baseUrl : '../../../Source',
+        waitSeconds : 60
+    });
+    </script>
+</head>
+<body class="sandcastle-loading" data-sandcastle-bucket="bucket-requirejs.html">
+<style>
+    @import url(../templates/bucket.css);
+</style>
+<div id="cesiumContainer" class="fullSize"></div>
+<div id="loadingOverlay"><h1>Loading...</h1></div>
+<div id="toolbar">
+</div>
+<script id="cesium_sandcastle_script">
+function startup(Cesium) {
+    'use strict';
+//Sandcastle_Begin
+var viewer = new Cesium.Viewer('cesiumContainer');
+
+// Used for Bounding Sphere Height Example
+var centerForSphere = new Cesium.Cartesian3();
+var earthUniformRadius = Cesium.Ellipsoid.WGS84.radii.x;
+var maxHeight = 15000000;
+var radius = earthUniformRadius + maxHeight;
+
+// The Bounding Sphere
+var boundingSphere = new Cesium.BoundingSphere(centerForSphere, radius);
+
+// View the Bounding Sphere
+var outlineSphere = viewer.entities.add({
+    name : 'Bounding Sphere Outline',
+    position: new Cesium.Cartesian3(0.01, 0.01, 0.01), // errors if 0,0,0
```

Comment:
Can we set this somewhere else then? 

Also like @hpinkos said, in general just limit the comments to things that are non-obvious.